src/lib/pubkey/pk_ops_impl.h

```diff
@@ -9,6 +9,9 @@
 #define BOTAN_PK_OPERATION_IMPL_H__
 
 #include <botan/pk_ops.h>
+#include <botan/eme.h>
+#include <botan/kdf.h>
```

Comment:
The reason for the empty destructors in pk_ops.cpp was to run ~unique_ptr<T> where T was forward declared (EMSA, KDF, EME), to avoid pulling in these headers. But, all three are public headers belonging to the basic API, and your change simplifies the code overall, so ok.